project_tag/__init__.py

```diff
@@ -1 +1,4 @@
+# Copyright 2019 Patrick Wilson <patrickraymondwilson@gmail.com>
```

Comment:
As per module template, https://github.com/OCA/maintainer-tools/tree/master/template/module, `__init__` files should contain only license